### ISSUE TITLE
ERROR LockLogin 1.0.4.3 waterfall 

### ISSUE THREAD

**Issue by thfree** [KarmaConfigs/page#16]
LockLogin-1.0.4.3.jar + waterfall-1.16-393.jar

![изображение](https://user-images.githubusercontent.com/13134495/105898162-2c1e2c80-602a-11eb-96a5-79c01b98a4e4.png)

```
[22:54:33 ERROR]: Task BungeeTask(sched=net.md_5.bungee.scheduler.BungeeScheduler@7baf6acf, id=2, owner=ml.karmaconfigs.lockloginsystem.bungeecord.Main@64d7b720, task=ml.karmaconfigs.lockloginsystem.bungeecord.utils.PluginManagerBungee$$Lambda$146/0x0000000840273040@107f4980, delay=0, period=0, running=true) encountered an exception
java.lang.NoClassDefFoundError: org/bukkit/plugin/java/JavaPlugin
        at ml.karmaconfigs.lockloginsystem.bungeecord.utils.PluginManagerBungee.lambda$doVersionCheck$1(PluginManagerBungee.java:262) ~[?:?]
        at net.md_5.bungee.scheduler.BungeeTask.run(BungeeTask.java:66) ~[waterfall-1.16-393.jar:git:Waterfall-Bootstrap:1.16-R0.4-SNAPSHOT:2eae08f:393]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: org.bukkit.plugin.java.JavaPlugin
        at net.md_5.bungee.api.plugin.PluginClassloader.loadClass0(PluginClassloader.java:63) ~[waterfall-1.16-393.jar:git:Waterfall-Bootstrap:1.16-R0.4-SNAPSHOT:2eae08f:393]
        at net.md_5.bungee.api.plugin.PluginClassloader.loadClass(PluginClassloader.java:37) ~[waterfall-1.16-393.jar:git:Waterfall-Bootstrap:1.16-R0.4-SNAPSHOT:2eae08f:393]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
        ... 5 more

```

**Comment by KarmaConfigs** [comment]
Will be fixed in 1.0.4.4

**Comment by KarmaConfigs** [comment]
Fixed: 

https://www.spigotmc.org/resources/gsa-locklogin.75156/update?update=383869
